### ISSUE TITLE
[Question] Can we get the pretrained projector weights for llava-7b-v1?

### ISSUE THREAD

**Issue by jihan-yin** [haotian-liu/LLaVA#158]
### Question

I've been running some experiments around llava-7b-v0 because the pretrained projector weights are not released for 7b-v1. However this has been problematic due to the tokenization issue with vicuna v0, resulting in improper masking of labels during finetuning. Can you make the projector weights available for us to experiment with?

**Comment by jihan-yin** [comment]
Thanks!
